lobby/src/test/java/org/triplea/server/http/spark/RegisteredUserLoginControllerTest.java

```diff
@@ -0,0 +1,45 @@
+package org.triplea.server.http.spark;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.triplea.http.client.lobby.login.LobbyLoginClient;
+import org.triplea.http.client.lobby.login.LobbyLoginResponse;
+import org.triplea.http.client.lobby.login.RegisteredUserLoginRequest;
+
+import feign.FeignException;
+
+class RegisteredUserLoginControllerTest extends SparkServerSystemTest {
+  private static final RegisteredUserLoginRequest REGISTERED_USER_LOGIN_REQUEST =
+      RegisteredUserLoginRequest.builder()
+          .name("Cannibals whine with death!")
+          .password("Scrawny desolations lead to the malaria.")
```

Comment:
I wonder where you get your filler texts from ^^

Reply:
https://plugins.jetbrains.com/plugin/7216-dummy-text-generator

I was using the latin for a while, but decided to stop as it gets highlighted for spelling errors.